--- ../../venv/lib/python3.10/site-packages/django_pivot/utils.py	2022-08-20 00:08:53.692656332 +0900
+++ ./utils.py	2022-08-19 23:39:56.257611126 +0900
@@ -1,5 +1,6 @@
 from django.core.exceptions import FieldDoesNotExist
 from django.utils.encoding import force_str
+import datetime
 
 
 def get_column_values(queryset, field, choices):
@@ -17,7 +18,16 @@
 
 
 def _database_choices(queryset, field):
-    return [(value, force_str(value)) for value in queryset.values_list(field, flat=True).distinct().order_by(field)]
+    values = queryset.values_list(field, flat=True).distinct().order_by(field)
+    result = []
+    for value in values:
+        newvalue = value
+        if type(value) == datetime.datetime:
+            newvalue = value.isoformat()
+        else:
+            newvalue = force_str(value)
+        result.append((value, newvalue))
+    return result
 
 
 def _get_field(model, field_names):
